src/internal/pfsdb/pfsdb.go

```diff
@@ -175,6 +176,33 @@ func Branches(db *pachsql.DB, listener col.PostgresListener) col.PostgresCollect
 	)
 }
 
+func ProjectKey(project *pfs.Project) string {
+	return project.Name
+}
+
+func Projects(db *pachsql.DB, listener col.PostgresListener) col.PostgresCollection {
+	return col.NewPostgresCollection(
+		projectsCollectionName,
+		db,
+		listener,
+		&pfs.ProjectInfo{},
+		nil,
```

Comment:
I do mildly wonder if it is worthwhile having a key check here, but I don’t feel it strongly.

Reply:
Unlike the other keys that have some structure like <repo>.<type>, and <repo@branch>, project is just <project>, which suggests that we don't need to enforce it's structure, besides having the user input sanitized